assets/built-in/transformers/kubernetes/operatorsfromtca/Dockerfile

```diff
@@ -1,4 +1,4 @@
 # FROM quay.io/konveyor/tca-service-api
 FROM quay.io/mihirc36/tackle-container-advisor
 
-COPY detect.py transform.py ./
+COPY detect.py transform.py catalogue.yaml ./
```

Comment:
How can users configure this catalogue?

Reply:
By using a custom transformer. They can replace it with their own catalogue.yaml

Reply:
users should be able to specify a different catalogue for each run, and that should not require changing the custom transformer image.

Reply:
done